src/main/resources/graphql/userSchema.graphqls

```diff
@@ -0,0 +1,11 @@
+type User {
+    id: ID!,
+    nickname: String!,
+}
+
+type Mutation {
+    createUser(userSaveDto: UserSaveRequestDto!): ID!
```

Comment:
[fix]
createUser have to return user's token because we don't have any signin process

Reply:
Ok, I'll change `ID!` to `String!`. In actually, above code works well.